kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,92 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+def get_pod_ready_status():
+    pods_status = runcommand.invoke("kubectl get pods -n litmus -o yaml")
+    yaml_pod = yaml.full_load(pods_status)
+    logging.info('len of pod items' + str(len(yaml_pod['items'])))
+    for pod in yaml_pod['items']:
+        return pod['status']['phase']
+
+
+# Install litmus and wait until pod is running
+def install_litmus():
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-v1.8.2.yaml")
```

Comment:
Not a need now, but it would be nice to have this somehow parametrized, because updates are quite often in litmus.

Reply:
Thoughts on adding the version of litmus into the configuration file to be able to update on your own but leave a working version in the master branch.